tutorials/marble/jupyterlab-troubleshooting.md

```diff
@@ -0,0 +1,16 @@
+# Troubleshooting
+
+## FAQ
+
+**My saved files are missing**
+
+Only files saved in the `writable-workspace` folder will be kept when the user exits JupyterLab or the server shuts down.
+
+Files saved in all other locations are not kept.
+
+
+**My git repository is missing**
```

Comment:
Is this not the same question as the one above?

Reply:
It is pretty much the same but it's something the user may notice and not make the connection.